src/account/UpgradeableModularAccount.sol

```diff
@@ -41,6 +42,8 @@ contract UpgradeableModularAccount is
     UUPSUpgradeable
 {
     using EnumerableSet for EnumerableSet.Bytes32Set;
+    using EnumerableSet for EnumerableSet.AddressSet;
+    using EnumerableMap for EnumerableMap.AddressToUintMap;
```

Comment:
Are `EnumerableSet.AddressSet` and `EnumerableMap.AddressToUintMap` used in this file? I think we no longer have any account storage fields of type `AddressSet`, and `AddressToUintMap` is only used for `pluginManifestHashes` iirc

Reply:
Gotcha! Removed in [a09ffb2](https://github.com/erc6900/reference-implementation/pull/90/commits/a09ffb20ee47ad8ab7f2287882dfa754bb236fc3)